Task_TheFlexGame/index.css

```diff
@@ -1 +1,60 @@
-/* Start coding below for FLEX GAME TASK*/
+/* Start coding below for FLEX GAME TASK*/

```

Comment:
There is a missing asterisk in the CSS comment syntax. It should be '/*' to start a comment.